pkg/materialize/format_specs.go

```diff
@@ -73,7 +73,7 @@ func GetFormatSpecStruc(v interface{}) SourceFormatSpecStruct {
 	}
 	if protobuf, ok := u["protobuf"]; ok && protobuf != nil && len(protobuf.([]interface{})) > 0 {
 		if csr, ok := protobuf.([]interface{})[0].(map[string]interface{})["schema_registry_connection"]; ok {
-			message := protobuf.([]interface{})[0].(map[string]interface{})["message_name"].(string)
+			message := protobuf.([]interface{})[0].(map[string]interface{})["message"].(string)
```

Comment:
Is this a bugfix? It's not immediately obvious where this is flowing from.

Reply:
Yep, seems like it has been there for a while 🤦 
https://github.com/MaterializeInc/terraform-provider-materialize/blob/68319193d1576420971d5e4cb95b60e8ab13fab3/pkg/resources/schema.go#L212-L217